modules/sdk/src/main/scala/org/tessellation/sdk/domain/healthcheck/consensus/HealthCheckConsensusRound.scala

```diff
@@ -105,7 +134,7 @@ class HealthCheckConsensusRound[F[_]: Spawn: Clock, K <: HealthCheckKey, A <: He
     }
 
   private def sendProposal: F[Unit] =
-    ownConsensusHealthStatus.flatMap(gossip.spread)
+    ownConsensusHealthStatus.flatMap(status => gossip.spreadCommon(status))
```

Comment:
Remember that `spreadCommon` is identified by content, so the content should be unique. Maybe add `roundId` to the message?

Reply:
there is a `roundId` and `owner` in the message, so it's unique for every round/peer